doc/whats_new.rst

```diff
@@ -72,6 +72,17 @@ Model Selection Enhancements and API Changes
     and :class:`model_selection.StratifiedShuffleSplit` is now renamed
     to ``n_splits``.
 
+  - Training scores and Timing information
+
+    The ``cv_results_`` arrays now include the training scores for each
+    cross-validation split (with keys such as ``'split0_train_score'``), as
+    well as their mean (``'mean_train_score'``) and standard deviation
+    (``'std_train_score'``).
```

Comment:
"To avoid the cost of evaluating training score, set `return_train_score=False`.